dotcom-rendering/src/web/components/Page.tsx

```diff
@@ -2,12 +2,55 @@ import React, { StrictMode } from 'react';
 import { EmotionCache } from '@emotion/cache';
 import { CacheProvider, Global, css } from '@emotion/react';
 import { focusHalo } from '@guardian/src-foundations/accessibility';
+import { from, until } from '@guardian/src-foundations/mq';
+import { labelStyles as adLabelStyles } from './AdSlot';
 
 type Props = {
 	children: React.ReactNode;
 	cache: EmotionCache;
 };
 
+const adStylesDynamic = css`
```

Comment:
@oliverlloyd Would it help a little if we clearly identified this as an anti-pattern?

```suggestion
// This way of injecting CSS is against the DCR principle and should
// be avoided. It is a temporary solution. Do not expand this list.
// TODO: Real AdSlot components
const adStylesDynamic = css`
```

Reply:
In my experience, when you add TODO comments saying something will be fixed in future, it is almost always never is. There are _six pages_ of [TODO comments in DCR](https://github.com/guardian/dotcom-rendering/search?q=TODO), most of them will continue to live there for years 😞